src/test/java/uk/gov/ons/ctp/response/action/service/ActionProcessingServiceTest.java

```diff
@@ -540,6 +547,41 @@ public void testCaseRefShouldBeOnActionRequest() throws CTPException {
     assertEquals("Case ref", captor.getValue().getCaseRef());
   }
 
+  @Test
+  public void testThatFieldCasesAreHandledAccordingly() throws Exception {
```

Comment:
Can this test name be more descriptive? At the moment you have to go line by line to work out the intent of the test.